src/variables/components/VariableCard.tsx

```diff
@@ -27,34 +29,37 @@ interface OwnProps {
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = OwnProps & ReduxProps
 
+@ErrorHandling
 class VariableCard extends PureComponent<
   Props & RouteComponentProps<{orgID: string}>
 > {
   public render() {
     const {variable, onDeleteVariable} = this.props
 
     return (
-      <ResourceCard
-        testID="resource-card variable"
-        contextMenu={
-          <VariableContextMenu
-            variable={variable}
-            onExport={this.handleExport}
-            onRename={this.handleRenameVariable}
-            onDelete={onDeleteVariable}
+      <ErrorBoundary>
```

Comment:
The decorator and error boundary component here seem redundant

Reply:
Removed decorator.